src/Corax/IndexWriter.cs

```diff
@@ -490,15 +490,33 @@ public IndexWriter([NotNull] Transaction tx, IndexFieldsMapping fieldsMapping) :
             Init();
         }
 
-        public class IndexEntryBuilder : IDisposable
+        public interface IIndexEntryBuilder
+        {
```

Comment:
What's the rationale behind the introduction of this interface?

Reply:
We need it for the training, we have two builders. One for training and one for indexing